test/unit/input/hello_world.wat

```diff
@@ -0,0 +1,11 @@
+(module
```

Comment:
heh, it's similar to the existing https://github.com/WebAssembly/binaryen/blob/master/test/hello_world.wat - kind of a minimal wasm. if wasm had strings, it could print hello world...